start-scada-platform-with-dashboard-sync.sh

```
@@ -0,0 +1,658 @@
+        "src/dashboard.py"
+    )
+
+    for candidate in "${dashboard_candidates[@]}"; do
+        if [[ -f "$candidate" ]]; then
+            # Verificar que contenga FastAPI, uvicorn o similar
+            if grep -q -E "(FastAPI|app.*=|@app\.|uvicorn|from fastapi)" "$candidate" 2>/dev/null; then
+                DASHBOARD_FILE="$candidate"
+                success "✅ Dashboard encontrado: $DASHBOARD_FILE"
+
+                # Verificar si usa puerto específico
+                local file_port=$(grep -oE "port.*[=:].*[0-9]{4,5}" "$candidate" | head -1 | grep -oE "[0-9]{4,5}" || true)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)